### PR TITLE
Restyle profile page

### PR DESCRIPTION
**Pull request by logan-r**
### Tickets:

- HCK-196

### List of changes:

- Update confirm email page so that it matches figma style document

### Type of change:

Please delete options that aren't relevant.

- [x] New feature (non-breaking change which adds functionality)

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [x] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### CODE REVIEW

File: src/features/Account/ConfirmationEmailSentComponent.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import * as React from 'react';
 import Helmet from 'react-helmet';
 
 import { APIResponse, Auth } from '../../api';
-import constructionSVG from '../../assets/images/construction.svg';
+import telescopeSVG from '../../assets/images/telescope.svg';
```
Comment (loreina) [inline src/features/Account/ConfirmationEmailSentComponent.tsx]: I think we just used the svg name for Profile page like `Bulby` so would be nice to stay consistent here

File: src/features/Account/ConfirmationEmailSentComponent.tsx [hunk 2]
```diff
@@ -48,17 +48,17 @@ class ConfirmationEmailSentComponent extends React.Component<
         <Flex alignItems={'center'} flexDirection={'column'}>
           <Box>
             <Image
-              src={constructionSVG}
-              imgHeight={'6rem'}
-              padding={'0.5rem'}
+              src={telescopeSVG}
+              imgHeight={'11rem'}
+              padding={'0 0 30px 0'}
```
Comment (loreina) [inline src/features/Account/ConfirmationEmailSentComponent.tsx]: was `32px` before, looks okay tho just a note

**Comment by pierreTklein** [comment]
Can you provide a screenshot for:

- mobile
- desktop

Thanks!

File: src/features/Account/ConfirmationEmailSentComponent.tsx (hunk 1 above)
Comment (loreina) [reply src/features/Account/ConfirmationEmailSentComponent.tsx]: also for images, we've been order imports by putting assets at the bottom in their own group, see `ManageAccountContainer.tsx`

**Comment by logan-r** [comment]
`StatusPage` was changed so that background image of city sky line doesn't render on confirm email page

Screenshots:
Desktop
Mobile

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Changed
+
+- Updated layout for confirm email page
+
+
```
Comment (loreina) [inline docs/CHANGELOG.md]: remove extra line here

**Pull request merged**